### PR TITLE
Add KSC info to the community page

### PR DESCRIPTION
**Pull request by terrytangyuan**
@kubeflow/kubeflow-steering-committee

### CODE REVIEW

File: content/en/docs/about/community.md [hunk 1]
```diff
@@ -103,6 +103,22 @@ In addition, please check out the community-curated [awesome list of projects an
 
 The Kubeflow trademark and logos are registered trademarks of Google, please review the [Kubeflow Brand Guidelines](https://github.com/kubeflow/community/blob/master/KUBEFLOW_BRAND_GUIDELINES.pdf) for more information.
 
+## Kubeflow Steering Committee
+
+The [Kubeflow Steering Committee (KSC)](https://github.com/kubeflow/community/blob/master/KUBEFLOW-STEERING-COMMITTEE.md) is the governing body of the Kubeflow project, providing decision-making and oversight pertaining to the Kubeflow project policies, sub-organizations, and financial planning, and defines the project values and structure.
```
Comment (james-jwu) [inline content/en/docs/about/community.md:108]: I think the table is duplicated from KUBEFLOW-STEERING-COMMITTEE.md? If so maybe a reference to the file would be sufficient.

Comment (terrytangyuan) [reply content/en/docs/about/community.md:108]: Good idea. Removed.

**Pull request merged**